tests/test_slack.py

```diff
@@ -10,16 +10,19 @@
 
 @pytest.fixture
 def mock_valid_slack_config():
+    """Mock valid Slack config."""
     return {"webhook": WEBHOOK}
 
 
 @pytest.fixture
 def mock_valid_slack_config_with_block_kit():
+    """Mock valid Slack config with block kit usage."""
     return {"webhook": WEBHOOK, "use_block_kit": True}
 
 
 @pytest.fixture
 def mock_invalid_slack_config():
+    """Mock invalid Slack config."""
```

Comment:
These are absolutely fine, and we can keep them, but I just wanted to mention that it's pretty normal for flake8-docstring to be configured to ignore test directories. But let's keep it like this. 👍🏼